index.d.ts

```diff
@@ -0,0 +1,25 @@
+export interface runJxa {
+    /**
+     * Returns a Promise for the value returned from input.
+     *
+     * @param {string | function} input - .
```

Comment:
Is `- .` a convention? I haven't seen that before. Are you sure the below lines will show up as description for `input`?